CHANGELOG.md

```diff
@@ -34,73 +34,20 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 # Changelog
 
-## [v10.0.0-rc0] 2023-05-19
+## [Unreleased]
 
-* (chore) Update module version to v10 [#2502](https://github.com/cosmos/gaia/pull/2502).
-* (deps) Bump multiple dependencies (sdk, cometbft, ibc-go, btcd), see [#2498](https://github.com/cosmos/gaia/pull/2498) for details.
-  * (deps) Bump [cosmos-sdk](https://github.com/cosmos/cosmos-sdk) to [v0.45.16-ics](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.45.16-ics). See [CHANGELOG.md](https://github.com/cosmos/cosmos-sdk/blob/release/v0.45.x/CHANGELOG.md) for details.
-  * (deps) Bump [ibc-go](https://github.com/cosmos/ibc-go) to [v4.4.0](https://github.com/cosmos/ibc-go/blob/release/v4.4.x/CHANGELOG.md). See [v4.2 Release Notes](https://github.com/cosmos/ibc-go/releases/tag/v4.4.0) for details.
-* (gaia) Bump Golang prerequisite from 1.18 to 1.20 see (https://go.dev/blog/go1.20) for details.
+## [v10.0.0] 2023-05-19
 
-## [v9.1.0] - 2023-05-08
```

Comment:
This release branch is based off v9.1.0, wouldn't it make sense for changelog readers to have context on the version this branch is built off?

Reply:
I think that every new major release would be based off the latest release, right? 

Also, to simplify the changelog of a release branch, we should just keep the versions related to that release branch.

Reply:
> I think that every new major release would be based off the latest release, right?

No typically a new major release branch is based off main. The v10 release branch deviates from main (it's based off v9.1.0), and that's why I think the v9 entries are relevant

Reply:
> we should just keep the versions related to that release branch.

Agreed, and in that case the v9 entries are relevant imo